package.json

```diff
@@ -52,6 +52,7 @@
     "multiaddr-to-uri": "^5.0.0",
     "prop-types": "^15.7.2",
     "pull-file-reader": "^1.0.2",
+    "queue": "^6.0.1",
```

Comment:
I missed this on the first pass: can we switch to [p-queue](https://github.com/sindresorhus/p-queue) instead?
We already use `p-queue` it in various other places across the stack, so it could be reused during the build + one less API to learn for potential contributor :)